readthedocs/templates/projects/domain_list.html

```diff
@@ -0,0 +1,53 @@
+{% extends "projects/project_edit_base.html" %}
+
+{% load i18n %}
+
+{% block title %}{% trans "Edit Domains" %}{% endblock %}
+
+{% block nav-dashboard %} class="active"{% endblock %}
+
+{% block editing-option-edit-proj %}class="active"{% endblock %}
+
+{% block project-domains-active %}active{% endblock %}
+{% block project_edit_content_header %}{% trans "Domains" %}{% endblock %}
+
+{% block project_edit_content %}
+  <p class="help_text">
+    {% trans "This allows you to add domains to your project. This allows them to live in the same namespace in the URLConf for a subdomain or CNAME." %}
+  </p>
+
+  <h3> {% trans "Existing Domains" %} </h3>
+  <p>
+    <ul>
+    {% for domain in object_list %}
+        <li {% if domain.machine %}class="help_text"{% endif %}>
+        {% if domain.canonical %}<strong>{% endif %}
+        {{ domain.url }}
+        {% if domain.canonical %}</strong>{% endif %}
+        (<a href="{% url 'projects_domains_edit' project.slug domain.pk %}">{% trans "Edit" %}</a>)
+        (<a href="{% url 'projects_domains_delete' project.slug domain.pk %}">{% trans "Remove" %}</a>)
+        </li>
+    {% endfor %}
+    </ul>
+  </p>
+  <h3> {% trans "Add new Domain" %} </h3>
+  <form method="post" action="{% url 'projects_domains_create' project.slug %}">{% csrf_token %}
+    {{ form.as_p }}
+    <p>
+      <input style="display: inline;" type="submit" value="{% trans "Submit" %}">
+    </p>
+  </form>
+{% endblock %}
+
+
+{% block footerjs %}
+      $('#id_domain').autocomplete({
+    source: '{% url "search_autocomplete" %}',
+    minLength: 2,
+    open: function(event, ui) {
+      ac_top = $('.ui-autocomplete').css('top');
+      $('.ui-autocomplete').css({'width': '233px', 'top': ac_top + 10 });
+    }
+  });
+
+{% endblock %}
```

Comment:
Ditto on this js block, a shared js block would be better than copying on multiple templates. This could probably be generalized as well, if its a common block of js that we need.